README.md

```diff
@@ -46,21 +60,51 @@ this.get('doStuff').perform();
 There's actually an even nicer syntax that's on the horizon, but
 is NOT YET SUPPORTED due to a [Babel bug](https://github.com/babel/babylon/issues/13).
 
+See the [**TypeScript**](#typescript) section to see it in action.
+
+When this Babel issue is addressed, this syntax should Just Work with
+this addon.
+
+##### Classic ("Ember Object Model") syntax with decorators
+
+Possible, but not recommended.
```

Comment:
How come this this "not recommended"? Is it just because it's the "old way" and things are generally gravitating towards ES6 class syntax and decorators?

Reply:
I had it somewhere in the back of my mind, that support for the object model was brittle / difficult before writing this PR. I had a chat with @pzuraq on Slack since then. Support for the object model was dropped in v2 of `ember-decorators`. 

The `@ember-decorators/utils` package still seems compatible with the object model (and the tests in here assert that), but I guess when [the proposal](https://github.com/tc39/proposal-decorators), *finally* [advances to stage 3](https://github.com/tc39/agendas/pull/398) and [Babel implements it](https://babeljs.io/docs/plugins/transform-decorators/), support for objects will be dropped, as the revised version of the proposal only allows decorators for classes.

I can add a note to the README, so people know.